src/mlpack/methods/rann/ra_model.hpp

```diff
@@ -18,13 +18,275 @@
 #include <mlpack/core/tree/cover_tree.hpp>
 #include <mlpack/core/tree/rectangle_tree.hpp>
 #include <mlpack/core/tree/octree.hpp>
-
+#include <boost/variant.hpp>
 #include "ra_search.hpp"
 
 namespace mlpack {
 namespace neighbor {
 
 /**
+ * Alias template for RASearch
+ */
+template<typename SortPolicy,
+         template<typename TreeMetricType,
+                  typename TreeStatType,
+                  typename TreeMatType> class TreeType>
+using RAType = RASearch<SortPolicy,
+                        metric::EuclideanDistance,
+                        arma::mat,
+                        TreeType>;
+
+template<typename SortPolicy>
+struct RAModelName
+{
+  static const std::string Name() { return "rank_approx_search_model";};
```

Comment:
I know that this is a really picky comment about style, but can you please add a space between the semicolon and closing brace?  I also think the final semicolon is unnecessary so we can just do `; }` at the end of the line.